ssl/pom.xml

```diff
@@ -101,10 +101,6 @@
             <groupId>org.wildfly.security</groupId>
             <artifactId>wildfly-elytron-base</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.wildfly.security</groupId>
-            <artifactId>wildfly-elytron-credential</artifactId>
```

Comment:
Can this be removed here? I think `SecurityDomainTrustManager` needs `org.wildfly.security.credential.X509CertificateChainCredential` so would have thought this would still be needed here but maybe I'm misremembering.